### PR TITLE
Random span-based API tests

### PR DESCRIPTION
**Pull request by MaggieTsang**
Added tests for Span API in Random class. References https://github.com/dotnet/corefx/issues/22356.

Depends on https://github.com/dotnet/coreclr/pull/13708.

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/System/Random.cs [hunk 1]
```diff
@@ -54,13 +54,35 @@ public static void Seeded()
         }
 
         [Fact]
-        public static void ExpectedValues()
+        public static void Seeded_Span()
```
Comment (danmosemsft) [inline src/System.Runtime.Extensions/tests/System/Random.cs]: Missing test passing a zero length span.

Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/System/Random.cs]: I expect you're going to need to separate this test out into its own file that's only built when targeting netcoreapp.  The test project has netstandard configurations, and the new API won't be available there.  Some examples are here: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj#L35-L44

**Comment by MaggieTsang** [comment]
@dotnet-bot test UWP NETNative x86 Release Build
@dotnet-bot test Packaging All Configurations x64 Debug Build

Couldn't delete workspace.

**Comment by danmosemsft** [comment]
Need to either baseline the uap errors or wait a few days for new project N bits. I suggest the latter.

**Comment by MaggieTsang** [comment]
@dotnet-bot test UWP NETNative x86 Release Build
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Pull request merged**